### DISCUSSION TITLE
GitLab Source: Request apps

### DISCUSSION THREAD

**Discussion by rumboalla** [rumboalla/apkupdater#477] [General]
Currently supported apps:
```kotlin
GitLabApp("com.aurora.store", "AuroraOSS", "AuroraStore")
```
If you want an app supported, please reply here using the format above (PackageName, GitLabUser, GitLabRepo) or send a Pull Request.

**Comment by HazakuraID** [comment]
ceno browser
Package: ie.equalit.ceno
```
https://gitlab.com/censorship-no/ceno-browser
```
